input/pagecontent/2-7-Conformance.md

```diff
@@ -9,13 +9,13 @@ Hubs SHOULD serve a JSON document at the location formed by appending `/.well-kn
 
 A simple JSON document is returned using the `application/json` mime type, with the following key/value pairs:
 
-Field | Optionality | Type | Description
---- | --- | --- | ---
-`eventsSupported` | Required | array | Array of FHIRcast events supported by the Hub.
-`websocketSupport` | Required | boolean | The static value: `true`, indicating support for websockets.
-`webhookSupport` | Optional | boolean | `true` or `false` indicating support for webhooks. Hubs SHOULD indicate their support for web hooks. 
-`fhircastVersion` | Optional | string | `STU1` or `STU2` or `STU3` indicating support for a specific version of FHIRcast. Hubs SHOULD indicate the version of FHIRcast supported. 
-`getCurrentSupport` | Optional | boolean | `true` or `false` indicating support for the "[Get Current Context](2-9-GetCurrentContext.html)" API. 
+{:.grid}
+Field              | Optionality | Type  | Description
+------------------ | ----------- | ----- | ---
+`eventsSupported`  | Required    | array | Array of FHIRcast events supported by the Hub.
+`websocketSupport` | Required    | boolean | SHALL have the static value: `true`, indicating support for websockets.

```

Comment:
Is it another backwards compatibility thing to keep websocketSupport in the conformance response?  Should it at least be optional for later deprecation?

Reply:
I think it's more compatible to hardcode it, as proposed. Suggest: we also don't error on encountering `webhookSupport`.